wayland-server/src/display.rs

```diff
@@ -134,7 +134,34 @@ impl DisplayHandle {
         self.handle.create_global::<State>(
             I::interface(),
             version,
-            Arc::new(GlobalData { data, _types: std::marker::PhantomData }),
+            Arc::new(GlobalData { data, _types: std::marker::PhantomData::<(_, _, State)> }),
+        )
+    }
+
+    /// Create a new protocol global
+    ///
+    /// This global will be advertized to clients through the `wl_registry` according to the rules
+    /// defined by your [`GlobalDispatch`] implementation for the given interface. Whenever a client
+    /// binds this global, the associated [`GlobalDispatch::bind()`] method will be invoked on your
+    /// `State`.
+    pub fn create_delegated_global<State, I, U, DelegateTo>(
```

Comment:
This documentation was copy/pasted without being updated

Reply:
My bad, addressed in https://github.com/Smithay/wayland-rs/pull/699/commits/ee9140d38bbf6922653196a3fbe1200b175808cc#diff-5e5b2f5127938323ccf49f8f4096fe02d2f630fe4338a4b345a79d42ff941a11R143-R147